src/olympia/ratings/models.py

```diff
@@ -86,7 +95,8 @@ class Rating(ModelBase):
         'self', null=True, related_name='reply', db_column='reply_to',
         on_delete=models.CASCADE)
 
-    rating = models.PositiveSmallIntegerField(null=True)
+    rating = models.PositiveSmallIntegerField(
+        null=True, choices=RATING_CHOICES)
```

Comment:
Adding `choices` prevents the admin from making an additional costly query to figure out the possible values for the filter.